src/reducers/member.js

```diff
@@ -17,16 +18,34 @@ export default function userReducer(state = initialState, action) {
       }
       return initialState;
     }
-    case 'USER_DETAILS_UPDATE': {
-      if (action.data) {
+    case 'USER_WISHLIST_REMOVE': {
+      if (action.removeLote) {
+        const wishlist = state.wishlist;
+
+        // lodash func removes from array by value
+        const newWishlist = remove(wishlist, (lote) => {
+          return lote !== action.removeLote;
```

Comment:
what is this condition about?

Reply:
this function goes through the wishlist and returns a new array with everything not equal to the lote we want to remove